### DISCUSSION TITLE
Stats are showing 0 commits. Top Language card works fine. Both default and own vercel instance.

### DISCUSSION THREAD

**Discussion by goodecodes** [anuraghazra/github-readme-stats#2126] [Q&A] [answered]
I am running my own vercel instance and PAT. I am able to display my Top Languages but my Stats Card shows 0 activity. I also ran into this issue when using the default instance. 
<img width="559" alt="Screen Shot 2022-10-04 at 7 59 01 PM" src="https://user-images.githubusercontent.com/114842849/193952115-4974725a-34a9-43da-9d9b-b29d7eac2f3d.png">

**Comment by goodecodes** [comment, marked as answer]
It was a simple issue, but my first encountering it. My commit email address wasn't set up to what was associated with my account so my pushes weren't registering. The commits I make are now show up after running `git config --global user.email "YOUR_EMAIL"`. Running `git config --global user.email` confirms the address you're commits are pushing with. Hope this helps anyone else with this issue.

**Reply by rickstaa** [reply]
@goodecoder great that you found a solution and thanks for sharing your answer.
